provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -680,6 +680,64 @@ spec:
                     minimum: 1
                     maximum: 1023
 ---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: droplogs.aci.droplog
+spec:
+  group: aci.droplog
+  names:
+    kind: Droplog
+    listKind: DroplogList
+    plural: droplogs
+    singular: droplog
+  scope: Cluster
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    schema:
+   # openAPIV3Schema is the schema for validating custom objects.
+      openAPIV3Schema:
+        type: object
+        properties:
+          apiVersion:
+            type: string
+          kind:
+            type: string
+          spec:
+            type: object
+            properties:
+              disableDefaultDropLog:
+                description: Disables the default droplog enabled by acc-provision.
+                default: false
+                type: boolean
+              dropLogMode:
+                default: unfiltered
+                type: string
+                enum:
+                  - unfiltered
+                  - flow-based
+              dropLogPruning:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    srcIP:
+                      type: string
+                    destIP:
+                      type: string
+                    srcMAC:
+                      type: string
+                    destMAC:
+                      type: string
+                    srcPort:
+                      type: integer
+                    destPort:
+                      type: integer
+                    ipProto:
```

Comment:
ipProto: is read as an integer from opflex.

Reply:
update it